src/Halogen/HTML/CSS.purs

```diff
@@ -43,9 +44,9 @@ instance stylesIsProp ∷ IsProp Styles where
 -- |                      display block ]
 -- |       [ ... ]
 -- | ```
-style ∷ ∀ i. CSS → Prop i
+style ∷ ∀ i r. CSS → HP.IProp (style ∷ String|r) i
 style =
-  prop (propName "style") (Just $ attrName "style")
+  HP.prop (HC.PropName "style")
```

Comment:
Instead of setting up a new `prop` and creating a `IsProp` instance for `Styles` I think we may as well reuse `HP.style` and compose in a function to print the `CSS` as a string. We can get rid of the `Styles` newtype then too, which was only there for the `IsProp` machinery.